src/actions/apiFetch.js

```diff
@@ -0,0 +1,51 @@
+import 'whatwg-fetch';
```

Comment:
why do we need that ?

Reply:
It's fetch polyfill

Reply:
I mean why we need polyfill here ? ) i think it is okey to support only evergreen browsers and do not support ie11 )